redhat/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM registry.access.redhat.com/rhel
+
+MAINTAINER Kong
+
+ENV KONG_VERSION 1.0.0rc2
+
+LABEL name="Kong" \
+      vendor="Kong" \
+      version="${KONG_VERSION}" \
+      release="1" \
```

Comment:
Should this be `revision`? Or is it a requirement that this label be named `release`? It is somewhat conflicting with the Kong release itself. "Revision" is, e.g., the terminology used for rockspecs.

Reply:
This stems from RedHat's official requirements on Dockerfiles - this `release` label is required.